rosdep/base.yaml

```diff
@@ -3294,6 +3297,7 @@ nodejs-legacy:
 npm:
   debian: [npm]
   fedora: [npm]
+  gentoo: [net-libs/nodejs]
```

Comment:
The useflags can't be specified here because of the portage syntax: unlike other package managers, you have to set a variable before calling the executable and this need a rework on how `rosdep` handles the package managers.
see https://bugs.gentoo.org/show_bug.cgi?id=621608